kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/continually.kt

```diff
@@ -29,10 +32,9 @@ suspend fun <T> continually(
    config: ContinuallyConfiguration<T>,
    test: suspend () -> T,
 ): T {
-
    delay(config.initialDelay)
 
-   val start = TimeSource.Monotonic.markNow()
+   val start = getContinuallyTimeSource().markNow()
```

Comment:
Just a style preference to consider as `get()` sounds rather imperative:
* top-level function `continuallyTimeSource()` or
* `ContinuallyTimeSource.current()`